inspirehep/modules/forms/static/js/forms/affiliations_typeahead.js

```diff
@@ -31,7 +31,7 @@ define([
     this.dataEngine = new Bloodhound({
       name: 'affiliations',
       remote: {
-        url: '/api/institutions?q=affautocomplete:%QUERY*',
+        url: '/api/institutions?q=affiliation_suggest:%QUERY*',
         filter: function(response) {
           return $.map(response.hits.hits, function(el) { return el });
```

Comment:
and the hits are probably under, `response.affiliation[0].options` instead `.hits.hits` thing.

Reply:
`return el.payload` maybe?

Reply:
And be sure everything inside https://github.com/inspirehep/inspire-next/pull/2795/files#diff-d5957147cc784ce12f5bc9c0855e7628R47 template is being populated https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/records/receivers.py#L335 .Otherwise they should be removed from the template or if necessary should be added to payload.

Reply:
do you know, why `map` is used here since `mapper` returns the element itself in the first place. @jmartinm ?